dockers/docker-sonic-mgmt-framework/start.sh

```diff
@@ -2,9 +2,3 @@
 
 mkdir -p /var/sonic
 echo "# Config files managed by sonic-config-engine" > /var/sonic/config_status
-
-rm -f /var/run/rsyslogd.pid
```

Comment:
Why this is removed everywhere?

Reply:
Because Guohan is now passing `-iNONE` to rsyslogd which disables writing a PID file. See http://man7.org/linux/man-pages/man8/rsyslogd.8.html

Reply:
add rsyslogd -iNONE so that the pid is no longer generated.

Reply:
Thanks